tests/modules/nf-core/circularmapper/circulargenerator/nextflow.config

```diff
@@ -0,0 +1,13 @@
+process {
+
+    publishDir = { "${params.outdir}/${task.process.tokenize(':')[-1].tokenize('_')[0].toLowerCase()}" }
+
+    withName: CIRCULARMAPPER_CIRCULARGENERATOR {
+        ext.args = { [
+            params.elongation_factor ? "-e ${params.elongation_factor}" : '-e 500',
```

Comment:
Are these required parameters to get the tool to run, with no default in the tool itself? 
We have a [policy of requiring mandatory inputs as an argument to the module](https://nf-co.re/docs/contributing/modules#migrating-from-pytest-to-nf-test).

Reply:
they don't have defaults and parameters are necesary